ros_ign_gazebo/launch/ign_gazebo.launch.py

```diff
@@ -30,9 +30,14 @@ def generate_launch_description():
     return LaunchDescription([
         DeclareLaunchArgument('ign_args', default_value='',
                               description='Arguments to be passed to Ignition Gazebo'),
+        # The default version for Foxy in Citadel (3)
+        DeclareLaunchArgument('ign_version', default_value='3',
```

Comment:
I can see us forgetting to update this when we make changes. Do you think it would be possible to inject the correct version here from CMake? Just asking in case it isn't too much hassle. Otherwise I think we can just port this to other branches now and hope we remember to do it when we branch off `ros2` for `humble` and others.